sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -217,6 +218,7 @@ case class CatalogTable(
     owner: String = "",
     createTime: Long = System.currentTimeMillis,
     lastAccessTime: Long = -1,
+    createVersion: String = "",
```

Comment:
actually, can we use current version as default value, and when reading table metadata in `HiveExternalCatalog`, set the version to the value from table properties or `Prior to 2.3` if no such entry in table properties.